README.md

```diff
@@ -51,24 +73,77 @@ Okay, lets get down to business.
 9. You may be asked to turn off and plug in your device multiple times, the script _should_ take care of the rest.
 
 ### Linux
+
 1. Either clone this repository or download it as ZIP
 2. Open a terminal in the directory of extracted files
-3. Run `./r1.sh`
-4. The script may ask you for your sudo permissions several times, so please enter your password
-5. You may be asked to turn off and plug in your device, the script _should_ take care of the rest.
+3. Download and unpack the `system.img` file containing modified AOSP 13.
+
+   ```bash
+   # Find a release here: https://github.com/RabbitHoleEscapeR1/r1_escape/releases
+   # Then set the RELEASE env variable:
+   RELEASE=20240605
+
+   # Download.
+   wget https://github.com/RabbitHoleEscapeR1/r1_escape/releases/download/${RELEASE}/system.img.xz
```

Comment:
This can be:
`wget https://github.com/RabbitHoleEscapeR1/r1_escape/releases/latest/download/system.img.xz`

In fact, this can possibly just be a check and download action in r1.sh and r1.ps1